index.js

```diff
@@ -106,15 +74,6 @@ const zooAnimals = [
   3. Return this new array
   */
 
-  function lowPopulationAnimals(array){
-    const lowPop = array.filter(animal => {
```

Comment:
Awesome job here. One cool think about the arrow functions is if you don't have more than one line you don't need the return statement as long as it is on one line (doesn't have curly braces)

```
const lowPopulationAnimals = zooAnimals.filter(
  (animal) => animal.population < 5,
);
```